webapp/src/store/workspace.ts

```diff
@@ -36,6 +44,13 @@ const workspaceSlice = createSlice({
 export const {setWorkspace} = workspaceSlice.actions
 export const {reducer} = workspaceSlice
 
+export const getUserWorkspaces = (state: RootState): UserWorkspace[] => state.workspace.userWorkspaces
+
+export const getUserWorkspaceList = createSelector(
+    getUserWorkspaces,
+    (userWorkspaces) => userWorkspaces,
+)
+
 export function getWorkspace(state: RootState): IWorkspace|null {
```

Comment:
Also, I think this should be `getCurrentWorkspace`.

Reply:
Done.